include/linux/tee_drv.h

```diff
@@ -169,6 +169,32 @@ int tee_device_register(struct tee_device *teedev);
  */
 void tee_device_unregister(struct tee_device *teedev);
 
+/**
+ * tee_session_calc_client_uuid() - Calculates client UUID for session
+ * @uuid:		Resulting UUID
+ * @connection_method:	Connection method for session
```

Comment:
Could be worth mentioning that accepted values are from `TEE_IOCTL_LOGIN_*`

Reply:
ok